app/assets/javascripts/directives/buddy_request_box.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
```

Comment:
I think you are right. I was using it for my IDE only. I should probably remove it.